firefly/app.py

```diff
@@ -39,11 +40,12 @@ def __init__(self, auth_token=None, allowed_origins=""):
         :param auth_token: the auto_token for the application
         :param allowed_origins: allowed origins for cross-origin requests
         """
+        self.name = name
         self.mapping = {}
         self.add_route('/', self.generate_index,internal=True)
         self.auth_token = auth_token
         self.allowed_origins = allowed_origins
-
+        self.flask_app = flask_app or Flask(name)
```

Comment:
Ideally it should be,
`self.flask_app = flask_app if flask_app else Flask(name)` as this is more readable.

The current statement also evaluates to the same result, however intent is not clear, and one has to know that Python evaluates `or` lazily.